### DISCUSSION TITLE
Possibly addressed issue! UpdateManager warning chain of infinite annoyance

### DISCUSSION THREAD

**Discussion by MarauderKnight3** [Pryaxis/TShock#2123] [General]
![Untitled](https://user-images.githubusercontent.com/48040746/94498632-7cd26580-01af-11eb-841e-19c70dd437bc.png)
How can I stop this from retrying?
Possibly, it started through `/checkupdates`.
This persists between server restarts.
Prerelease 12, 4.4.0.0.
I didn't put this in issues because I thought this may have been looked at already.

**Comment by hakusaro** [comment]
Oh no. That's a problem I have to fix.

**Comment by hakusaro** [comment]
@MarauderKnight3 thank you for bringing this to my attention! The issue has been fixed and you should no longer see this error. Please let me know if you continue to see this error.

**Reply by hakusaro** [reply]
The issue tracking this is https://github.com/Pryaxis/TShock/issues/2124. Please comment there if you definitely still see it.
